Documentation/tutorial/links_and_cross_references.md

```diff
@@ -0,0 +1,269 @@
+
+```markdown
+[file2](/subfolder/file2.html)
+```
+
+Sometimes you may find it's complicated to calculate relative path between two files.
+DocFX also supports path starts with `~` to represent path relative to the root directory of your project (i.e. where `docfx.json` is located).
+This kind of path will also be validated and resolved during build. For example, in the above case, you can write the following links in `file2.md`:
+ 
+```markdown
+[file1](~/file1.md)
+
+[file1](../file1.md)
+```
+
+Both will resolve to `../file1.html` in output html.
+
+> [Automatic link](https://daringfireball.net/projects/markdown/syntax#autolink) doesn't support relative path. If you write something like `<file.md>`, it will be treated as an HTML tag rather than a link.
+
+### Links in file includes
+
+If you use [file include](../spec/docfx_flavored_markdown.md#file-inclusion) to include another file, the links in included file is relative to the included file. For example, if `file1.md` includes `file2.md`:
+
+```markdown
+[!include[file2](subfolder/file2.md)]
+```
+
+All links in `file2.md` are relative to the `file2.md` itself, even when it's included by `file1.md`.
+
+Please note the file path in include syntax are handled differently than Markdown link. You can only use relative path to specify location of the included file. And it's not required included file to be included in `docfx.json`.
+
+> Each file in `docfx.json` will build into an output file. But included files usually don't need to build into individual topics. So it's not recommended to include them in `docfx.json`.
+
+### Links in inline HTML
+
+Markdown supports to [inline HTML](https://daringfireball.net/projects/markdown/syntax#html). DocFX also supports to use relative path in inline HTML. Path in HTML link (`<a>`), image (`<img>`), script (`<script>`) and css (`<link>`) will also be resolved if they're relative path.
+
+## Using cross reference
+
+Besides using file path to link to another file, DocFX also allows you to give a file an ID so that you can reference this file using that ID instead of its file path. This is useful in the some cases:
+
+1. Path to file is long and difficult to memorize or changes frequently.
+2. API reference documentation is usually auto generated so it's difficult to find its file path.
+3. You want to reference to files in another project without need to know its file structure.
+
+The basic syntax for cross referencing a file is:
+
+```markdown
+<xref:id_of_another_file>
+```
+
+This is similar to [automatic link](https://daringfireball.net/projects/markdown/syntax#autolink) syntax in Markdown but with a `xref` scheme. This link will build into:
+
+```html
+<a href="path_of_another_file">title_of_another_file</a>
+```
+
+You can see one benefit of using cross reference is you don't need to specify the link text and DocFX will automatically resolve it for you.
```

Comment:
Consider changed to:
As you can see, one benefit of using cross reference is that ...